src/base/io/readCbModel.m

```diff
@@ -353,10 +353,11 @@
                     end
                 end
             end
-        catch
+        catch ME
             % IF we are here, there was a problem in verifyModel or convertOldStyleModel, so this is
             % not a model.
-    end
+            0;
```

Comment:
Why is there a `0`?

Reply:
Debugging... 
Needed a breakpoint, so I needed some code.
Removed it.